### ISSUE TITLE
Scheduled/Draft post: Triggering subscriber emails and Publicize shares despite not being published

### ISSUE THREAD

**Issue by pauljacobson** [Automattic/wp-calypso#48911]
Our customer reported that a draft/scheduled post triggered emails to subscribers, along with a Publicize share to LinkedIn, despite the post not having been published. The user was using the block editor.

I'm opening this issue in case the issue occurs again for this, or another customer.

#### Steps to reproduce the behavior

1. The user prepared a draft post, and scheduled it for publication on 11 January 2021.
2. The user noticed an email for the post had been sent before the post was published.
3. The user also noticed that the post has been shared on LinkedIn at the same time.
4. Clicking on the link to the post produced a _Not Found_ page, confirming that the post was not public at the time.
5. When the post was published at the scheduled time and date, it did not trigger emails to subscribers or a Publicize share.

#### What I expected to happen

Draft or scheduled posts should not trigger emails to subscribers or Publicize shares until they are published.

#### What actually happened

A draft/scheduled post triggered emails to subscribers, and was shared to LinkedIn.

#### Context

Ticket: 3635379-zen

### Browser / OS version

Unknown

### Is this specific to the applied theme? Which one?

I don't have any information about this. The user's theme is a retired theme, Baskerville.

### Does this happen on simple or atomic sites or both?

This was a free site. I haven't experienced this issue previously.

### Is there any console output or error text?

Not applicable

### Level of impact (Does it block purchases? Does it affect more than just one site?)

Potentially a one-off issue.

**Comment by Robertght** [comment]
I found the opposite report here: https://github.com/Automattic/jetpack/issues/10651

I'm not able to find another similar case though. @jeherve is this more Jetpack-related than WordPress.com?

**Comment by jeherve** [comment]
Yes, that would be Jetpack-related, although I am not able to reproduce this issue at the moment.

I'll transfer this to the Jetpack repository.
